CHANGELOG.md

```diff
@@ -1,5 +1,99 @@
 
+## [4.3.00](https://github.com/kokkos/kokkos-kernels/tree/4.3.00) (2024-03-19)
+[Full Changelog](https://github.com/kokkos/kokkos-kernels/compare/4.2.01...4.3.00)
+
+### New Features
+
+#### BLAS updates
+- Syr2 [\#1942](https://github.com/kokkos/kokkos-kernels/pull/1942)
+
+#### LAPACK updates
+- Creation of the 'lapack' subdirectory, parallel to 'blas' [\#1985](https://github.com/kokkos/kokkos-kernels/pull/1985)
+- Lapack - SVD [\#2092](https://github.com/kokkos/kokkos-kernels/pull/2092)
+
+#### Sparse updates
+- Add sptrsv execution space overloads [\#1982](https://github.com/kokkos/kokkos-kernels/pull/1982)
+- Add block support to all SPILUK algorithms [\#2064](https://github.com/kokkos/kokkos-kernels/pull/2064)
+- Add support for BSR matrices to some trsv routines [\#2104](https://github.com/kokkos/kokkos-kernels/pull/2104)
+- GMRES: Add support for BSR matrices [\#2097](https://github.com/kokkos/kokkos-kernels/pull/2097)
+- Spmv handle [\#2126](https://github.com/kokkos/kokkos-kernels/pull/2126)
+- Option to apply RCM reordering to extracted CRS diagonal blocks [\#2125](https://github.com/kokkos/kokkos-kernels/pull/2125)
+
+#### ODE updates
+- ODE: BDF methods [\#1930](https://github.com/kokkos/kokkos-kernels/pull/1930)
+
+#### Misc updates
+- Add HIPManagedSpace support [\#2079](https://github.com/kokkos/kokkos-kernels/pull/2079)
+
+### Enhancements:
+
+#### BLAS
+- Axpby: improvement on unification attempt logic and on the execution of a diversity of situations [\#1895](https://github.com/kokkos/kokkos-kernels/pull/1895)
+
+#### LAPACK
+- Lapack - SVD: adding quick return when cuSOLVER is skipped [\#2107](https://github.com/kokkos/kokkos-kernels/pull/2107)
+
+#### Sparse
+- Refactor Test_Sparse_sptrsv [\#2102](https://github.com/kokkos/kokkos-kernels/pull/2102)
+
+#### Misc updates
+- Use execution space operator== [\#2136](https://github.com/kokkos/kokkos-kernels/pull/2136)
+
+#### TPL support
+- Add TPL support for KokkosBlas::dot [\#1949](https://github.com/kokkos/kokkos-kernels/pull/1949)
+- Add CUDA/HIP TPL support for KokkosSparse::spadd [\#1962](https://github.com/kokkos/kokkos-kernels/pull/1962)
+- Async matrix release for MKL >= 2023.2 in SpMV [\#2074](https://github.com/kokkos/kokkos-kernels/pull/2074)
+- Don't call optimize_gemv for one-shot MKL spmv [\#2073](https://github.com/kokkos/kokkos-kernels/pull/2073)
+
+### Build System:
+- Support CUBLAS_{LIBRARIES,LIBRARY_DIRS,INCLUDE_DIRS,ROOT} and KokkosKernels_CUBLAS_ROOT CMake options [\#2075](https://github.com/kokkos/kokkos-kernels/pull/2075)
+- Link std::filesystem for IntelLLVM in perf_test/sparse [\#2055](https://github.com/kokkos/kokkos-kernels/pull/2055)
+- Fix Cuda TPL finding [\#2098](https://github.com/kokkos/kokkos-kernels/pull/2098)
+- CMake: error out in certain case [\#2115](https://github.com/kokkos/kokkos-kernels/pull/2115)
+- Kokkos Kernels: change the default offset ETI from size_t to int [\#2140](https://github.com/kokkos/kokkos-kernels/pull/2140)
```

Comment:
@lucbv said #2140 should not be in this release unless the corresponding Trilinos changes are trivial, but it will take a bit of work and testing to update the matrix offset type in several places.

At today's meeting we think the plan is to use KokkosKernels's ``default_size_type`` for this instead of ``size_t`` - that way, the user can control the type based on KokkosKernels ETI options (but int is the new default, maximizing TPL availability).

Reply:
@brian-kelley thanks for the catch, I'll revert that PR in the release candidate and update the changelog

Reply:
Updated release-candidate-4.3.00 to revert #2140 and updated the changelog accordingly to drop the entry,  [aeb0dce](https://github.com/kokkos/kokkos-kernels/pull/2148/commits/aeb0dcee876c9a4773ab8a8e76b90d37a77b0899)